### PR TITLE
 setup.cfg: Allow non-coverage of sys.exit(main())

### PR DESCRIPTION
**Pull request by jayvdb**
Related to https://github.com/coala/coala/issues/4166

Split off from https://github.com/coala/coala/pull/4169

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `corobo mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: setup.cfg [hunk 1]
```diff
@@ -67,3 +67,4 @@ exclude_lines =
   pragma ${PLATFORM_SYSTEM}: no cover
   pragma ${OS_NAME}: no cover
   pragma Python [0-9.,]*${PYTHON_VERSION}[0-9.,]*: no cover
+  sys.exit\(main\(\)\)
```
Comment (RaiVaibhav) [inline setup.cfg]: We can have a alternate way, check this out https://medium.com/@george.shuklin/how-to-test-if-name-main-1928367290cb

Comment (jayvdb) [reply setup.cfg]: I've explained on https://gitlab.com/coala/mobans/issues/84 why I think that is unnecessary.

If necessary, we have a check script which verifies that all entry points defined in the yaml have exactly these two lines at end of the file.  That prevents these sequences from being different in multiple entry points, and also ensures they are identical on all repos with entry points.

**Comment by jayvdb** [comment]
ack https://github.com/coala/coala/pull/5680/commits/9ad7b102cee20ced8f10cb1d35f6cc442b1c3d20 https://github.com/coala/coala/pull/5680/commits/d54c67edbaf07f5e99d535addf02fcfe5bdbddd6  https://github.com/coala/coala/pull/5680/commits/cdb60a71368185bb2a0a2936ac86eb3a3950f511

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Comment by jayvdb** [comment]
@gitmate-bot ff

**Pull request merged**